lib/valkyrie/sequel/metadata_adapter.rb

```diff
@@ -33,32 +31,13 @@ def id
       end
     end
 
-    def perform_migrations!(drop: false)
+    def perform_migrations!
       Sequel.extension :migration
-      drop_database! if drop
-      create_database!
       Sequel::Migrator.run(connection, "#{__dir__}/../../../db/migrations")
     end
 
     def reset_database!
```

Comment:
why do you still need this if it only calls `perform_migrations!`?

Reply:
Because I wanted it to roll back and retry migrations - forgot about that. Just added it.